From b3fb72308942f705e3e8e17765e4097773ed5483 Mon Sep 17 00:00:00 2001
From: Akshay S Dinesh <asdofindia@gmail.com>
Date: Mon, 19 Jun 2017 09:44:50 +1000
Subject: [PATCH xkeyboard-config] typofix tel-sarala key layout

This was causing error in loading new keyboard description for all
using Indian keyboards

Signed-off-by: Akshay S Dinesh <asdofindia@gmail.com>
---
 symbols/in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/symbols/in b/symbols/in
index 7658ee7..79d5099 100644
--- a/symbols/in
+++ b/symbols/in
@@ -1472,7 +1472,7 @@ xkb_symbols "tel-sarala"
     key <AC09> { [          U0C32,          U0C33,          U0C0C,          U0C61 ] }; // ల ళ ఌ ౡ
     key <AC10> { [          U0C1F,          U0C20,      semicolon,          colon ] }; // ట ఠ ; :
     key <AC11> { [     quoteright,       quotedbl	    	    		  ] }; // ' " 
-    key <AD01> { [          U0C46,          U0C0E,          U0C44           U0C34 ] }; // ె ఎ ౄ ఴ
+    key <AD01> { [          U0C46,          U0C0E,          U0C44,          U0C34 ] }; // ె ఎ ౄ ఴ
     key <AD02> { [          U0C38,          U0C37,          U0C44                 ] }; // స ష ౄ
     key <AD03> { [          U0C47,          U0C0F,          U0C44                 ] }; // ే ఏ ౄ
     key <AD04> { [          U0C30,          U0C31,          U0C44,          U0C60 ] }; // ర ఱ ౄ ౠ
-- 
2.13.5

